### PR TITLE
Add requirements for Open edX Redwood

### PR DESCRIPTION
**Pull request by mrtmm**

### CODE REVIEW

**Review by fghaas — COMMENTED** [review]
If I understand correctly, Tutor 18 now runs Open edX on Python 3.11. I think that means we should update the test matrix to include that version, correct?

https://github.com/overhangio/tutor/blob/v18.1.1/tutor/templates/build/openedx/Dockerfile#L27

**Comment by mrtmm** [comment]
Yep that makes sense, let's see if I added it correctly.

**Comment by fghaas** [comment]
> Yep that makes sense, let's see if I added it correctly.

Almost. :) You still need to add 3.11 to the build matrix in `.github/workflows/tox.yml`.

File: tox.ini [hunk 1]
```diff
@@ -1,9 +1,10 @@
 [tox]
-envlist = py38-{lilac,maple,nutmeg,olive,palm,quince},flake8,report
+envlist = py{38,311}-{lilac,maple,nutmeg,olive,palm,quince,redwood},flake8,report
 
 [gh-actions]
 python =
-    3.8: py38-lilac,py38-maple,py38-nutmeg,py38-olive,py38-palm,py38-quince,flake8
+    3.8: py38-lilac,py38-maple,py38-nutmeg,py38-olive,py38-palm,py38-quince,py38-redwood,flake8
+    3.11: py311-lilac,py311-maple,py311-nutmeg,py311-olive,py311-palm,py311-quince,py311-redwood,flake8
```
Comment (fghaas) [inline tox.ini:7]: I'd argue that this ought to be sufficient:
```suggestion
    3.8: py38-lilac,py38-maple,py38-nutmeg,py38-olive,py38-palm,py38-quince,flake8
    3.11: py311-redwood,flake8
```

File: tox.ini [hunk 2]
```diff
@@ -1,9 +1,10 @@
 [tox]
-envlist = py38-{lilac,maple,nutmeg,olive,palm,quince},flake8,report
+envlist = py{38,311}-{lilac,maple,nutmeg,olive,palm,quince,redwood},flake8,report
```
Comment (fghaas) [inline tox.ini:2]: ... and we could possibly reduce the matrix here, too.

```suggestion
envlist = py38-{lilac,maple,nutmeg,olive,palm,quince},py311-redwood,flake8,report
```

**Review by fghaas — APPROVED** [review]
Looks good to me now, thanks!

**Pull request merged**